docs/sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,56 @@
 Version history
 ===============
 
+5.4.0 (2017 March 21)
+---------------------
+
+File format improvements:
+
+* DICOM
+   - added support for DICOMDIR files, which allow multiple DICOM files in a single 
+     directory to be opened as a single dataset
+   - plane position values for values X, Y and Z are now being set in OME-XML
+* Nikon NIS-Elements ND2
+   - performance improvements based on reading chunkmap. Processing of the chunkmap 
+     can be disabled via the MetadataOptions API using the boolean option 
+     ``nativend2.chunkmap``. For ImageJ users this option can be accessed via a checkbox 
+     in the Nikon ND2 section of the Bio-Formats configuration dialog 
+     :menuselection:`Plugins --> Bio-Formats --> Bio-Formats Plugins Configuration` (thanks to Christian Sachs)
+* OME-TIFF
+   - added an option to save an OME-TIFF dataset as a binary TIFF and companion XML. 
+     This can be used via the bfconvert command line tool by setting the value of 
+     option ``ometiff.companion`` to the name of the companion file to use. For example 
+     ``bfconvert -option ometiff.companion outputFile.companion.ome inputFile.tiff outputFile.ome.tiff``
+
+Bug fixes:
```

Comment:
I would just leave the TiffSaver point under this heading and list all the rest under file format improvements together